app/ts/library/identifyTokens.ts

```diff
@@ -0,0 +1,47 @@
+import { Contract, Provider, Interface, ZeroAddress, BytesLike } from "ethers";
+import { ERC721ABI, MulticallABI } from "./abi.js";
+
+export async function detectNft(address: string, id: bigint, provider: Provider) {
+	const multicall = new Contract('0x5ba1e12693dc8f9c48aad8770482f4739beed696', MulticallABI, provider)
+	const nftInterface = new Interface(ERC721ABI)
+
+	const calls = [
```

Comment:
doesn't ethers have nicer way to call functions instead of using `encondeFunctionData`?